source/Assets/Scripts/InfinarioSDK/Infinario.cs

```diff
@@ -15,6 +15,11 @@ private Infinario()
 		{
 			//Prepare for wrappers
 			implementation = new SDK.Unity ();
+
+			//Setup decimal separator for numbers
+            var customCulture = (System.Globalization.CultureInfo)System.Threading.Thread.CurrentThread.CurrentCulture.Clone();
+            customCulture.NumberFormat.NumberDecimalSeparator = ".";
+            System.Threading.Thread.CurrentThread.CurrentCulture = customCulture;    
```

Comment:
You can't change a thread's culture from within the SDK, there are two reasons for that:
* You may affect the host application, which may have set their own culture
* The SDK runs in multiple threads and you are changing the culture for only one of them

Please use the correct Culture explicitly at the place where the string gets formatted. You want to use `CultureInfo.InvariantCulture` as the culture as it already has `.` as  the decimal separator. I'm not sure about thousands separator though, please check that.